easybuild/easyconfigs/m/MATIO/MATIO-1.5.8-foss-2016a.eb

```diff
@@ -0,0 +1,21 @@
+easyblock = 'ConfigureMake'
+
+name = 'MATIO'
+version = '1.5.8'
+
+homepage = 'http://sourceforge.net/projects/matio/'
+description = """matio is an C library for reading and writing Matlab MAT files."""
+
+toolchain = {'name': 'foss', 'version': '2016a'}
+
+source_urls = [SOURCEFORGE_SOURCE]
+sources = [SOURCELOWER_TAR_GZ]
+
+dependencies = [('zlib', '1.2.8')]
+
+sanity_check_paths = {
+    'files': ['include/matio.h', 'bin/matdump', 'libmatio.a'],
```

Comment:
@RvDijk sanity check failed on missing `libmatio.a`, should be `lib/libmatio.a`?

It would be nice to add this custom `sanity_check_paths` to the existing MATIO easyconfigs too BTW (in a separate PR), are you up for that?